### PR TITLE
support abstract class

### PR DESCRIPTION
**Pull request by iamdennisme**

### CODE REVIEW

**Comment by vickumar1981** [comment]
:+1:   lgtm.

Thanks for this.

File: src/jupyterdartkernel/kernel.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def runCode(self, command):
             curr_line = lines_read[idx]
             if curr_line.startswith("import "):
                 import_lines.append(curr_line)
-            elif curr_line.startswith("class "):
+            elif curr_line.startswith("class ") or curr_line.startswith("abstract "):
```
Comment (vickumar1981) [inline src/jupyterdartkernel/kernel.py:77]: anything else that we should possibly pull out and move to the front of the file?

Comment (iamdennisme) [reply src/jupyterdartkernel/kernel.py:77]: mixin and extension

Comment (vickumar1981) [reply src/jupyterdartkernel/kernel.py:77]: @iamdennisme   how do those work in dart?  extension methods are cool.  would love to add support for that.  suppose, i can google it, but an example would be great.

**Pull request merged**